.gitignore

```diff
@@ -32,7 +32,7 @@ test/testdata/**/hie.yaml
 .shake/
 
 # pre-commit-hook.nix
-.pre-commit-config.yaml
+#.pre-commit-config.yaml
```

Comment:
Is there a way to put the config into nix setting?

Reply:
I don't use nix so I have no idea, any help would be appreciative.

Reply:
nix doesn't support windows so I suppose it is not necessary?

Reply:
Oh, makes sense.